docs/how-to/troubleshooting-omnitrace-linux.rst

```diff
@@ -0,0 +1,73 @@
+.. meta::
```

Comment:
Do we have the post-install section pointing to anything here?
Still would prefer this to be merged in post-install

Reply:
I'll add this to the post-install section and we can review. I'll leave the original guide for now in case we change our mind or prefer the original wording. Leaving as unresolved until we decide.